pkg/doc/jose/jwk.go

```diff
@@ -213,6 +215,22 @@ func (j *JWK) MarshalJSON() ([]byte, error) {
 	return (&j.JSONWebKey).MarshalJSON()
 }
 
+// KeyType returns the kms KeyType of the JWK, or an error if the JWK is of an unrecognized type.
+func (j *JWK) KeyType() (kms.KeyType, error) {
```

Comment:
@Baha-sk here's the new method, taking the logic out of `decorator.AttachmentData.Verify()`

Reply:
it can be improved to check if jwk.Key is ECDSA keys .. it can be IEEE1363Type keys for ecdsa keys